src/engine/bank01.asm

```diff
@@ -7172,7 +7172,45 @@ Func_6cfa: ; 6cfa (1:6cfa)
 	ret
 ; 0x6d3f
 
-	INCROM $6d3f, $6d84
+Func_6d3f: ; 6d3f (1:6d3f)
+	or a
+	bit 7, [hl]
+	ret z
+	push hl
+	bit 6, [hl]
+	ld a, $0a
```

Comment:
This hard-coded value `$0a` is better seen as the value `10`. It is the damage received due to poison (and likewise for the `$14`/`20` below).
There should be constants created for these values. These new constants probably would belong in `src/constants/duel_constants.asm` near `PSN_DBLPSN` unless someone else has a different suggestion.